modules/host/persist.go

```diff
@@ -201,27 +201,3 @@ func (h *Host) load() error {
 	}
 	return nil
 }
-
-// initPersist loads all of the saved host state into the host object. If there
-// is no saved state, suitable defaults are chosen instead.
-func (h *Host) initPersist() error {
-	// Create the perist directory if it does not yet exist.
-	err := os.MkdirAll(h.persistDir, 0700)
-	if err != nil {
-		return err
-	}
-
-	// Initialize the logger. Logger must be initialized first, because the
-	// rest of the initialization makes use of the logger.
-	h.log, err = persist.NewLogger(filepath.Join(h.persistDir, logFile))
-	if err != nil {
-		return err
-	}
-
-	// Load the prior persistance structures.
-	err = h.load()
-	if err != nil {
-		return err
-	}
-	return nil
-}
```

Comment:
moved initPersist into New, flattening it. I think you asked me to do this in a much earlier PR, but I have only recently decided that it's the right thing to do.